src/SFA.DAS.EmployerIncentives.Functions.AcceptanceTests/SFA.DAS.EmployerIncentives.Functions.AcceptanceTests.csproj

```diff
@@ -5,6 +5,7 @@
   </PropertyGroup>
 
   <ItemGroup>
```

Comment:
Don't think we need this section.  Please delete.

Reply:
Removed